tutorials/HT/partitioned-heat/fenics-fenics/heat.py

```diff
@@ -173,10 +171,10 @@ def fluxes_from_temperature_full_domain(F, V):
 
 if problem is ProblemType.DIRICHLET:
```

Comment:
could this go into `Adapter.initialize`?

Reply:
This is probably something we should discuss. There are good reasons for both possible ways.

Reply:
Maybe this discussion also belongs here https://github.com/precice/fenics-adapter/issues/3